docs/product/visualizations.md

```diff
@@ -11,111 +11,85 @@ tags: [product]
 Visualizations have two main purposes:
 
 - **Display results of nodes**  
-  Each node can be assigned with one or more visualization. After a node
-  computes its new value, the visualization shows it in an understandable way to
-  the user. Please note that a single node can be assigned with multiple
-  visualizations at the same time. For example, a node might want to display a
-  map of locations, and their list at the same time next to each other.
+  Each node can be assigned with one or more visualization. After a node computes its new value, the visualization shows
+  it in an understandable way to the user. Please note that a single node can be assigned with multiple visualizations
+  at the same time. For example, a node might want to display a map of locations, and their list at the same time next
+  to each other.
```

Comment:
This and below paragraphs were extended to 120 chars per line. Please revert it back to 100